wp-content/civi-extensions/goonjcustom/goonjcustom.php

```diff
@@ -127,6 +130,35 @@ function goonjcustom_evaluate_tokens(TokenValueEvent $e) {
       }
 
       $row->tokens('contact', 'inductionDetails', $inductionDetailsMarkup);
+			// Fetch induction activity details
+			$inductionActivity = \Civi\Api4\Activity::get(FALSE)
+					->addSelect('activity_date_time', 'Induction_Fields.Goonj_Office')
+					->addWhere('activity_type_id:name', '=', 'Induction')
+					->addWhere('source_contact_id', '=', $contactId)
+					->setLimit(1)
+					->execute()
+					->first();
```

Comment:
Use single everywhere on this PR wherever needed

Reply:
sure